src/query/query_dump.c

```diff
@@ -3274,6 +3293,21 @@ qdump_print_stats_text (FILE * fp, xasl_node * xasl_p, int indent)
   qdump_print_stats_text (fp, xasl_p->scan_ptr, indent);
   qdump_print_stats_text (fp, xasl_p->connect_by_ptr, indent);
 
+  if (xasl_p->sq_cache && XASL_IS_FLAGED (xasl_p, XASL_USES_SQ_CACHE))
+    {
+      fprintf (fp, "%*c", indent, ' ');
+      if (xasl_p->sq_cache->enabled)
+	{
+	  fprintf (fp, "SUBQUERY_CACHE (hit: %d, miss: %d, size: %luB, status: enabled)\n",
```

Comment:
When I tested again, the output of the "B" character made it less readable.
How about explaining in the manual that the unit is bytes and not printing the letter "B"?

There is no unit output in JSON format output.
There is no unit output in time output.

Reply:
Okay.